modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/json_serializable/enum.mustache

```diff
@@ -0,0 +1,8 @@
+{{#description}}/// {{{description}}}{{/description}}
+enum {{{enumName}}} {
+{{#allowableValues}}
+{{#enumVars}}
+  {{{name}}},
```

Comment:
enum parameter can have a value, so field `FIELD_1` can be represented by `field-1` and so `field-1` must be mapped to `FIELD_1`

Reply:
This supported from 4.2.0 https://pub.dev/documentation/json_annotation/latest/json_annotation/JsonEnum/fieldRename.html

Reply:
Latest commits have fixed up generation of enums, and I've used `@JsonValue()` to specify the enum values